PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -154,62 +154,29 @@ static int __stdcall BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM, LPARAM pDa
 
 generic_string folderBrowser(HWND parent, const generic_string & title, int outputCtrlID, const TCHAR *defaultStr)

```

Comment:
Do we need to convert `getFolderName()` to use the same CustomFileDialog?
That function is defined after this one.

Reply:
Yes, please! 👍 
Or better : just replace it by `folderBrowser()` if it is it possible.

Reply:
Now `getFolderName()` just calls `folderBrowser()`. It behaves like a shortcut with a predefined parameters like the title.